wgsl/index.bs

```diff
@@ -876,22 +996,33 @@ array_decoration
 # Variable and const # {#variables}
 
 TODO: *Stub* (describe what a constant is): A constant is a name for a value, declared via a `const` declaration.
+What types are permitted?  Storable, plus pointer to store type.
+
+TODO(dneto): A const may not be of type pointer-to-handle. A function parameter may not have type pointer-to-handle.
+Otherwise we'd have a need to make a pointer-to-handle type expression. But we've reserved the `handle` keyword.
+When translating from SPIR-V, you must trace through
+the OpCopyObject (or no-index OpAccessChain) instructions that might be between
+the pointer-to-array and the pointer-to-struct.
 
 A <dfn dfn noexport>variable</dfn> is a named reference to storage that can contain a value of a
 particular storable type.
 
-Two types are associated with a variable: its *store type* (the type of value
-that may be placed in the referenced storage) and its *reference type* (the type
-of the variable itself).  If a variable has store type *T* and storage class *S*,
+Two types are associated with a variable: its <dfn noexport>store type</dfn> (the type of value
```

Comment:
it's not clear to me what this refers to. Maybe some examples are needed?

Reply:
Intent of this MR was to add cross-links, that's all.

Store-type and reference-type will become significant when description of how pointers are type-checked and evaluated. I'd like to defer that until later.